evap/staff/tests/test_views.py

```diff
@@ -1114,7 +1114,7 @@ def helper_publish_evaluation_with_publish_notifications_for(self, evaluation, c
         page = self.app.get("/staff/semester/1", user=self.manager)
         form = page.forms["evaluation_operation_form"]
         form['evaluation'] = evaluation.pk
-        response = form.submit('target_state', value="published")
+        response = form.submit('target_state', value=Evaluation.state_to_str(Evaluation.State.PUBLISHED))
```

Comment:
Your next comment is actually where this string is handled. The strings are converted to / used to look up these operation classes which call the transition methods on an evaluation and these transitions have the actual `Evaluation.State` values already replaced. I had left these as strings, because I thought that they would be "user-facing" in a URL, but they could of course also be integers. What do you think?

Reply:
I wouldn't mind if the operations were integers. I mean, our courses and semesters and everything is integers too in urls. But that's something for @janno42 to decide i would say.

Reply:
yes, please use the integers. people are not meant to fiddle with urls manually anyway :)